src/main/java/com/xeeva/catalog/CostCenterPage.java

```diff
@@ -1,532 +1,583 @@
+	 * @author: Praveen Namburi, @version: Created 22-09-2016.
+	 * @param daysOut
+	 */
+	public void change_RequiredByAtHeaderLevel(){
+		// after cart check out, application taking time to load cost center page 
+		lblCC.syncVisible(30, false);
+		//To click on calender at CC_HeaderLevel.
+		driver.executeJavaScript("arguments[0].click();", lnkDateCC_HeaderLvel);
+		List<WebElement> nextMonthArrows = driver.findElements(By.xpath("html/body/div[@class='calendar']/"
+				+ "table/thead/tr[2]/td[4]"));
+		int loopCount=0;
+		for(WebElement nextMonth : nextMonthArrows){
+			nextMonth.click();
+			driver.setElementTimeout(Constants.ELEMENT_TIMEOUT);
+			List<WebElement> selectDates = driver.findElements(By.xpath("html/body/div[@class='calendar']/"
+					+ "table/tbody/tr[4]/td[@class='day']"));
+			for(WebElement selWeekDate : selectDates){
+				selWeekDate.click();
+				//driver.setPageTimeout(5);
+				loopCount++;
+				break;
+			}
+			if(loopCount>0) break;
+		}
+
+		//Verify the message -'The RequiredBy date has been updated' is displayed.
+		verify_RequiredByDateUpdated_atCCLevel();
+		//driver.setElementTimeout(4);
+
+		//Get the RequiredBy date at header level.
+		String getReqByDate_AtHeaderLevel = txtReqByHeaderLevel.getText();
+		TestReporter.log("Get ReqByDate_before Change: " + getReqByDate_AtHeaderLevel);
+
+		List<WebElement> inputDates = dateCCInputLineLevel;
+		int getInputDatesSize = inputDates.size();
+		TestReporter.log("Total no. of input dates available at CC_line-level: " + getInputDatesSize);
+		//Iterate through each ReqBydate element and get text,
+		//verify the expected date with actual values.
+		for(WebElement inputDate : inputDates){
+			String getModifiedDate = inputDate.getAttribute("value");
+			TestReporter.assertEquals(getReqByDate_AtHeaderLevel, getModifiedDate, 
+					"Date has been updated at line-level for the id: - ["+inputDate.getAttribute("id")+"]");
+		}
+
+	}
+
+
+	/**
+	 * @summary Method To click button - Shop For More Items 
+	 * @author  Lalitha Banda
+	 * @date    28/09/16
+	 */
+	public void click_ShopForMoreItems(){
+		pageLoaded();
+		btnShopForMoreItems.syncVisible(5, false);
+		btnShopForMoreItems.jsClick();
+	}
+
+
+	/**
+	 * @summary Method to verify Edit link Enabled or Disabled in costcenter page
+	 * @author  Praveen Namburi
+	 * @date    27/09/16
+	 */
+	public void verifyEDITlinkEnabledOrNot(){
+		// Verify Edit for Non Price agreement 
+		List<WebElement> unitPriceEdits = driver.findElements(By.xpath(".//*[@id='customfa']/tbody/tr/td[6]/div/input"));
+		for(WebElement input : unitPriceEdits){
+			String randomID =input.getAttribute("name").replaceAll("\\D+", "");
+			String classValue = driver.findElement(By.xpath(".//*[@id='trCCRow_"+randomID+"']/td[10]/a[2]/i")).getAttribute("class");
+			if(classValue.contains("cursor-pointer")){
+				TestReporter.assertTrue(true, "Edit Item for non Price Agreement Item is Enabled !!");
```

Comment:
Is the purpose of this method to verify that the edit links are enabled?  If they are not enabled, do you need the test to fail?  If so it needs to be done like so:
TestReporter.assertTrue(classValue.contains("cursor-pointer"), "Verify the edit item for non price agreement is enabled");